cli/src/test/resources/ide-projects/intellij/repository/intellij/intellij/default/linux/bin/idea.sh

```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
 cd "$(dirname "$0")"
+echo $PWD
+echo "intellij linux $*"
```

Comment:
Oh, that `echo $PWD` was also for debugging on github.
We can discuss if we remove it here or add it to all these scripts.
I would not block this PR from merging because of this since I really want to finish the release that we post-poned multiple times already...